vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -26888,7 +27684,65 @@ unit:PicoPA-PER-KiloM
   qudt:applicableSystem sou:CGS-GAUSS ;
   qudt:applicableSystem sou:SI ;
   qudt:conversionMultiplier 1.0e-15 ;
-  qudt:hasDimensionVector qkdv:A0E0L0I0M1H0T-2D0 ;
+  # unit:PicoPA-PER-KiloM has incorrect dimension vector:
+  # 	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T-2D0, which should be
+  # 	http://qudt.org/vocab/dimensionvector/A0E0L-2I0M1H0T-2D0
+  # factor unit tree for unit pPa/km
+  #     +-- unit:PicoPA-PER-KiloM         	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T-2D0
+  #     |   +-- unit:PicoPA                   	http://qudt.org/vocab/dimensionvector/A0E0L-1I0M1H0T-2D0
+  #     |   |   +-- unit:M^-2                     	http://qudt.org/vocab/dimensionvector/A0E0L-2I0M0H0T0D0
+  #     |   |   +-- unit:N                        	http://qudt.org/vocab/dimensionvector/A0E0L1I0M1H0T-2D0
+  #     |   |   |   +-- unit:SEC^-2                   	http://qudt.org/vocab/dimensionvector/A0E0L0I0M0H0T-2D0
+  #     |   |   |   +-- unit:M                        	http://qudt.org/vocab/dimensionvector/A0E0L1I0M0H0T0D0
+  #     |   |   |   +-- unit:KiloGM                   	http://qudt.org/vocab/dimensionvector/A0E0L0I0M1H0T0D0
+  #     |   +-- unit:KiloM^-1                 	http://qudt.org/vocab/dimensionvector/A0E0L-1I0M0H0T0D0
+  #
+  # the unit has these quantityKinds: EnergyPerArea, ForcePerLength
+  #  ══╤ qk:EnergyPerArea
+  #    ╞═══ qk:RadiantExposure
+  #    ╞═══ qk:SurfaceTension
+  #    ├─── unit:BTU_IT-PER-FT2
+  #    ├─── unit:FT-LB_F-PER-FT2
+  #    ├─── unit:FT-LB_F-PER-M2
+  #    ├─── unit:GigaJ-PER-M2
+  #    ├─── unit:J-PER-CentiM2
+  #    ├─── unit:J-PER-M2
+  #    ├─── unit:KiloBTU_IT-PER-FT2
+  #    ├─── unit:KiloCAL-PER-CentiM2
+  #    ├─── unit:KiloGM-PER-SEC2
+  #    ├─── unit:KiloW-HR-PER-M2
+  #    ├─── unit:MegaJ-PER-M2
+  #    ├─── unit:N-M-PER-M2
+  #    ├─── unit:PicoPA-PER-KiloM
+  #    ├─── unit:W-HR-PER-M2
+  #    └─── unit:W-SEC-PER-M2
+  #  ══╤ qk:ForcePerLength
+  #    ╞══╤ qk:LinearForce
+  #    │  └─── unit:KiloN-PER-M
+  #    ╞═══ qk:LinearStiffness
+  #    ├─── unit:DYN-PER-CentiM
+  #    ├─── unit:KiloGM_F-M-PER-CentiM2
+  #    ├─── unit:KiloLB_F-PER-FT
+  #    ├─── unit:LB_F-PER-FT
+  #    ├─── unit:LB_F-PER-IN
+  #    ├─── unit:MilliN-PER-M
+  #    ├─── unit:N-M-PER-M2
+  #    ├─── unit:N-PER-CentiM
+  #    ├─── unit:N-PER-M
+  #    ├─── unit:N-PER-MilliM
+  #    └─── unit:PicoPA-PER-KiloM
+  # quantity kinds that fit the corrected dimension vector http://qudt.org/vocab/dimensionvector/A0E0L-2I0M1H0T-2D0, and associated units:
+  #  ══╤ qk:PressureLossPerLength
+  #    └─── unit:KiloGM-PER-M2-SEC2
+  #  ══╤ qk:SpectralRadiantEnergyDensity
+  #    ├─── unit:J-PER-M4
+  #    ├─── unit:KiloPA-PER-MilliM
+  #    └─── unit:PA-PER-M
+  #  ══╤ qk:ModulusOfSubgradeReaction
+  #    └─── unit:N-PER-M3
+  #  ══╤ qk:SpecificWeight
+  #    └─── unit:KiloN-PER-M3
```

Comment:
setting QK to qk:SpectralRadiantEnergyDensity as it has another Pressure-per-length unit